test/parallel/test-stream-transform-constructor-set-methods.js

```diff
@@ -18,7 +24,8 @@ function _flush(n) {
 
 const t = new Transform({
   transform: _transform,
-  flush: _flush
+  flush: _flush,
```

Comment:
We should use `common.mustCall()` on these as well, which should also allow us to avoid the unnecessary boolean checks on exit.